api/handlers_test.go

```diff
@@ -1244,6 +1244,132 @@ func TestServer_JetDropsByID(t *testing.T) {
 	})
 }
 
+func TestServer_JetDropsByJetID(t *testing.T) {
```

Comment:
it will be nice to check `sort_by` param (because now its not working and tests still pass)